docs/openapi/components/schemas/common/DeliverySchedule.yml

```diff
@@ -215,76 +215,8 @@ example: |-
     },
     "injectionDate":"2022-05-01  10:13:00 PM",
     "deliveryDate":"2022-05-01  10:13:00 PM",
-    "portOfEntry":{
-      "geo":{
-        "type":[
-          "GeoCoordinates"
-        ],
-        "latitude":"40.6936",
-        "longitude":"-89.589"
-      },
-      "type":[
-        "Place"
-      ],
-      "address":{
-        "type":[
-          "PostalAddress"
-        ],
-        "postalCode":"",
-        "addressRegion":"IL",
-        "streetAddress":"",
-        "addressCountry":"USA",
-        "addressLocality":"Peoria"
-      }
-    },
-    "portOfDestination":{
-      "type":[
-        "Place"
-      ],
-      "globalLocationNumber":"4775333880136",
-      "geo":{
-        "type":[
-          "GeoCoordinates"
-        ],
-        "latitude":"-19.6752",
-        "longitude":"141.8303"
-      },
-      "address":{
-        "type":[
-          "PostalAddress"
-        ],
-        "name":"Waelchi - Emard",
-        "streetAddress":"25775 Moen Shores",
-        "addressLocality":"East Cielofort",
-        "addressRegion":"Alabama",
-        "postalCode":"00548-1233",
-        "addressCountry":"Andorra"
-      }
-    },
-    "portOfArrival":{
-      "type":[
-        "Place"
-      ],
-      "globalLocationNumber":"2040450460298",
-      "geo":{
-        "type":[
-          "GeoCoordinates"
-        ],
-        "latitude":"-20.0464",
-        "longitude":"17.9557"
-      },
-      "address":{
-        "type":[
-          "PostalAddress"
-        ],
-        "name":"Mraz Group",
-        "streetAddress":"6892 Kozey Trail",
-        "addressLocality":"Leannonland",
-        "addressRegion":"Georgia",
-        "postalCode":"48950-9803",
-        "addressCountry":"French Polynesia"
-      },
-      "unLocode":"DKCPH"
-    },
+    "portOfEntry": "3501",
+    "portOfDestination": "3901",
+    "portOfArrival": "4105",
```

Comment:
where do we find these codes?

Reply:
Given these are CBP port codes, this credential should probably be titled CBPDeliverySchedule... since nobody else can use it.

https://www.cbp.gov/sites/default/files/assets/documents/2020-Jan/ACE%20Appendix%20E%20Schedule%20D%20January%202020%20%281%29.pdf

Reply:
Or just maybe `USACBPDeliverySchedule`, since there's no guarantee other nations will never have their own CBP and/or `CBPDeliverySchedule`